src/app/core/market/api/bid/bid.model.ts

```diff
@@ -27,25 +30,52 @@ export class Bid {
 
   setFilter() {
     this.orderItems = [];
-    const orders = [];
+    this.buyOrders = [];
```

Comment:
build a new model for all this shizzle; this is starting to differentiate too much from the market api.